openff/interchange/components/_packmol.py

```diff
@@ -889,7 +889,7 @@ def solvate_topology_nonwater(
     solvent: Molecule,
     padding: Quantity = 1.2 * unit.nanometer,
     box_shape: NDArray = RHOMBIC_DODECAHEDRON,
-    target_density: Quantity = 1.0 * unit.gram / unit.milliliter,
+    target_density: Quantity = 0.6 * unit.gram / unit.milliliter,
```

Comment:
Unlike `solvate_topology`, `solvate_topology_nonwater` could have a lot of different solvents with a lot of different densities. I agree 0.6 g/mL is a better default for organic solvents than 1.0 g/mL, but I think making this a required argument is a better option. 

I designed the API of the `solvate_topology` function to allow the user not to think about the box, because a very reasonable default does exist, and overwriting any single part of that default makes sense for some special case (and I got sick of writing the same calculation in every prep workflow).

With a generalized solvent, that default doesn't really exist - the padding distance might need to be different if finite size effects are more or less important, and the density is straightforwardly unknown. In the case of padding I think the "images of the solute don't see each other's VDWs" heuristic is still at least well grounded, but the density is really a mystery. The best case is that the system blows up and a user thinks "why is that the default". The worst case is we get the density only slightly wrong, the box equilibrates, the padding distance is just made meaningless and some finite size effect is discovered after publication.

So I'd suggest

```suggestion
    target_density: Quantity,
```

Reply:
You make a good case